ocs_ci/ocs/ocp.py

```diff
@@ -841,6 +841,78 @@ def get_logs(
         return output
 
 
+def get_provider():
```

Comment:
shouldn't this function be in `ocs_ci/ocs/node.py`

Reply:
moved.